jetstream/external_config.py

```diff
@@ -54,10 +54,13 @@ def from_github_repo(cls) -> "ExternalConfigCollection":
 
         for file in files:
             if file.name.endswith(".toml"):
-                slug = os.path.splitext(file.name)[0]
-                spec = AnalysisSpec.from_dict(toml.loads(file.decoded_content.decode("utf-8")))
-                last_modified = parser.parse(str(file.last_modified))
-                configs.append(ExternalConfig(slug, spec, last_modified))
+                commits = repo.get_commits(path=file.path)
+
+                if commits.totalCount:
+                    slug = os.path.splitext(file.name)[0]
+                    spec = AnalysisSpec.from_dict(toml.loads(file.decoded_content.decode("utf-8")))
+                    last_modified = parser.parse(str(commits[0].commit.author.date))
```

Comment:
I think we might want committer and not author for the date to reflect when it landed in the repo.